requirements.bazel.txt

```diff
@@ -38,3 +38,4 @@ typing-extensions==4.9.0
 pyasn1-modules==0.3.0
 zipp==3.17.0
 wrapt==1.16.0
+typeguard==3.0.2
```

Comment:
3.0.2 was released last March, we should consider using the latest version.

Reply:
updated